tools/regression_tests/src/web/assets/jquery.js

```diff
@@ -0,0 +1,5 @@
+/*! jQuery v1.11.3 | (c) 2005, 2015 jQuery Foundation, Inc. | jquery.org/license */
```

Comment:
It can but it's very unsafe and leads to issues if you don't have a network connection for some reason. We already ship a few others in data/js/ for the same reasons I assume.